tests/k8s_test.go

```diff
@@ -182,7 +182,7 @@ func testNodeCreationFromConfig(t *testing.T, mngr, ty, name string, fields ...s
 
 /* -- test creation of single resource -- */
 func TestK8sClusterNode(t *testing.T) {
-	testNodeCreation(t, nil, nil, k8s.Manager, "cluster", k8s.ClusterName)
+	testNodeCreation(t, nil, nil, k8s.Manager, "cluster", "cluster")
```

Comment:
use `k8s.Manager` instead of `"cluseter"`

Reply:
Done